### PR TITLE
SDK-81 catch all exceptions caused by SDK stats

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**

### CODE REVIEW

File: maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/StatsManager.java [hunk 1]
```diff
@@ -32,16 +32,20 @@ private void loadStatistics() throws MojoExecutionException {
...
+                if (StringUtils.isNotBlank(goal)) {
+                    sdkStatistics.incrementGoal(goal);
+                    sdkStatistics.setLastUsed();
+                    sdkStatistics.save();
+                }
+                sdkStatistics.sendReport(wizard);
             }
-            sdkStatistics.sendReport(wizard);
+        } catch (Exception e) {
+            wizard.showMessage("Failed to send anonymous user statistics");
```
Comment (rkorytkowski) [inline maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/StatsManager.java]: Can you append at least e.getMessage() to give any clue to the user why?

Comment (AdamGrzybkowski) [inline maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/StatsManager.java]: Done

File: maven-plugin/src/main/resources/sdk-stats.properties [hunk 2]
```diff
@@ -10,6 +10,9 @@ statsDeleteCalls=0
 statsDeployCalls=0
 statsHelpCalls=0
 statsInfoCalls=0
+statsPrCalls=0
+statsReleaseCalls=0
+statsCloneCalls=0
```
Comment (rkorytkowski) [inline maven-plugin/src/main/resources/sdk-stats.properties]: Did you add those to the google form and post message?

Comment (AdamGrzybkowski) [inline maven-plugin/src/main/resources/sdk-stats.properties]: No, thanks for reminding ;) I did also forget to add this to the POST request. I will add this and create PR

**Pull request merged**